### PR TITLE
Add io.github.JakubMelka.Pdf4qt

### PR DESCRIPTION
**Pull request by JakubMelka**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by JakubMelka** [comment]
bot, build io.github.JakubMelka.Pdf4qt

**Comment by JakubMelka** [comment]
Hello, I've successfully built a Flatpak locally and it functions as expected. Would it be possible to enable network access during the build process? I utilize the VCPKG package manager and require it to download and build dependencies. Thank you for your assistance.

**Comment by hfiguiere** [comment]
> Would it be possible to enable network access during the build process?

No. it is not possible.

File: .appdata.xml
Comment (hfiguiere) [inline .appdata.xml:1]: this file should be submitted and managed upstream for future releases

Also please name it properly. File that start with a period are considered hidden.

File: .appdata.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <category>Office</category>
+    <category>Utility</category>
+  </categories>
+  <developer_name>Jakub Melka</developer_name>
+  <url type="homepage">https://jakubmelka.github.io/</url>
+  <url type="bugtracker">https://github.com/JakubMelka/PDF4QT/issues</url>
+  <url type="help">https://github.com/JakubMelka/PDF4QT/discussions</url>
+  <update_contact>Mgr.Jakub.Melka_AT_gmail.com</update_contact>
+  <content_rating type="oars-1.1"/>
+  <launchable type="desktop-id">io.github.JakubMelka.Pdf4qt.Pdf4QtDocDiff.desktop</launchable>
```
Comment (hfiguiere) [inline .appdata.xml:24]: only one launchable.

File: io.github.JakubMelka.Pdf4qt.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "cleanup": [
+        "/pdf4qt",
+        "/vcpkg"
+    ],
+    "modules": [
+        {
+            "name": "PDF4QT",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./build_pdf4qt.sh"
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:26]: you can't do that. This script want to download things. This is NOT possible.

If you need any dependency they should be either sources for this module, or separate modules installed.

File: io.github.JakubMelka.Pdf4qt.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                {
+                    "type": "archive",
+                    "url": "https://github.com/JakubMelka/PDF4QT/archive/refs/tags/v1.3.5.tar.gz",
+                    "sha256": "18f908cffc4e303b3866aa12545708126068fadeea0afaea42aff4f9094e481f",
+                    "dest": "pdf4qt"
+                },
+                {
+                    "type": "file",
+                    "path": "build_pdf4qt.sh",
+                    "dest-filename": "build_pdf4qt.sh"
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:38]:
```suggestion
                    "path": "build_pdf4qt.sh"
```

File: io.github.JakubMelka.Pdf4qt.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                },
+                {
+                    "type": "file",
+                    "path": ".appdata.xml",
+                    "dest-filename": ".appdata.xml"
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:43]:
```suggestion
                    "path": ".appdata.xml"
```

File: build_pdf4qt.sh [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+mv -f /app/share/applications/Pdf4QtViewerLite.desktop /app/share/applications/io.github.JakubMelka.Pdf4qt.Pdf4QtViewerLite.desktop
+mv -f /app/share/applications/Pdf4QtViewerProfi.desktop /app/share/applications/io.github.JakubMelka.Pdf4qt.Pdf4QtViewerProfi.desktop
+
+mv -f /app/share/icons/Pdf4QtDocDiff.svg /app/share/icons/io.github.JakubMelka.Pdf4qt.Pdf4QtDocDiff.svg
+mv -f /app/share/icons/Pdf4QtDocPageOrganizer.svg /app/share/icons/io.github.JakubMelka.Pdf4qt.Pdf4QtDocPageOrganizer.svg
+mv -f /app/share/icons/Pdf4QtViewerLite.svg /app/share/icons/io.github.JakubMelka.Pdf4qt.Pdf4QtViewerLite.svg
+mv -f /app/share/icons/Pdf4QtViewerProfi.svg /app/share/icons/io.github.JakubMelka.Pdf4qt.Pdf4QtViewerProfi.svg
+
+# Rename icons in desktop files
+sed -i 's|Pdf4QtDocDiff.svg|io.github.JakubMelka.Pdf4qt.Pdf4QtDocDiff.svg|g' /app/share/applications/io.github.JakubMelka.Pdf4qt.Pdf4QtDocDiff.desktop
```
Comment (hfiguiere) [inline build_pdf4qt.sh:57]: use `desktop-file-edit` to fix the desktop files. sed failure more is inexistant (and obscure)

File: build_pdf4qt.sh [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+rmdir /app/usr/share/icons
+rmdir /app/usr/share/applications
+rmdir /app/usr/share
+rmdir /app/usr
+
+# Edit the files
+
+sed -i 's|/usr/bin/||g' /app/share/applications/*.desktop
+sed -i 's|/usr/share/icons/||g' /app/share/applications/*.desktop
```
Comment (hfiguiere) [inline build_pdf4qt.sh:42]: if you reference these path is you desktop files, then the desktop files are wrong.

File: build_pdf4qt.sh [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+cd ..
+export VCPKG_ROOT=$(pwd)/vcpkg
+
+# Install tbb
+${VCPKG_ROOT}/vcpkg install tbb
+
+# Build
+cd pdf4qt
+export CMAKE_BUILD_PARALLEL_LEVEL=$(nproc)
+cmake -B build -S . -DCMAKE_BUILD_TYPE=Release -DPDF4QT_INSTALL_QT_DEPENDENCIES=0 -DCMAKE_TOOLCHAIN_FILE=${VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake -DCMAKE_INSTALL_PREFIX=/app
```
Comment (hfiguiere) [inline build_pdf4qt.sh:17]: the module should just be `buildsystem: cmake-ninja`

File: .appdata.xml
Comment (JakubMelka) [reply .appdata.xml:1]: I am author of PDF4QT. I can move it to the upstream. Should it be named io.github.JakubMelka.PDF4QT?

File: .appdata.xml (hunk 1 above)
Comment (JakubMelka) [reply .appdata.xml:24]: I have question. There are four applications (and a single command line tool). I would like to have all of them in the desktop menu. It works, when I built it locally. How to solve it?

File: build_pdf4qt.sh [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+./bootstrap-vcpkg.sh -disableMetrics
+cd ..
+export VCPKG_ROOT=$(pwd)/vcpkg
+
+# Install tbb
+${VCPKG_ROOT}/vcpkg install tbb
+
+# Build
+cd pdf4qt
+export CMAKE_BUILD_PARALLEL_LEVEL=$(nproc)
```
Comment (hfiguiere) [inline build_pdf4qt.sh:16]: it's NEVER nproc. We have `FLATPAK_BUILDER_N_JOBS` for that. Using nproc make it impossible to set the number of parallel task with flatpak-builder.

See my comment about using `buildsystem: cmake-ninja`

File: build_pdf4qt.sh [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+# Move icons and desktop files to the right place
+cd ..
+
+mkdir -p /app/lib/pdf4qt
+mkdir -p /app/share/icons
+mkdir -p /app/share/applications
+mkdir -p /app/share/metainfo
+
+mv -f /app/lib/*Plugin.so*  /app/lib/pdf4qt/
+mv -f /app/usr/share/icons/*.svg /app/share/icons/
```
Comment (hfiguiere) [inline build_pdf4qt.sh:30]: if you the build system install stuff in `usr` like that, it's wrong.

File: io.github.JakubMelka.Pdf4qt.json (hunk 2 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json:26]: So, is it not possible to use the standard VCPKG package system in some way? Would it be acceptable to prepare binary compiled dependent libraries into a separate GitHub repository, and then download and use them?

Comment (hfiguiere) [reply io.github.JakubMelka.Pdf4qt.json:26]: if VCPKG list what it download and from where, then maybe, like an offline mode. There are solution for cargo (Rust), pip (Python), go, etc.  See https://github.com/flatpak/flatpak-builder-tools So someone need to contribute VCPKG support.

Or you can add the dependencies in the manifest. Like other packages do.

File: build_pdf4qt.sh [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+mv -f /app/usr/share/applications/*.desktop /app/share/applications/
+mv -f .appdata.xml /app/share/metainfo/io.github.JakubMelka.Pdf4qt.appdata.xml
+
+rmdir /app/usr/share/icons
+rmdir /app/usr/share/applications
+rmdir /app/usr/share
+rmdir /app/usr
```
Comment (hfiguiere) [inline build_pdf4qt.sh:37]: this is what `cleanup` rules are for.

File: build_pdf4qt.sh (hunk 6 above)
Comment (JakubMelka) [reply build_pdf4qt.sh:42]: Desktop files are prepared for standard environments (like creating a .deb package) for which they are functional. However, Flatpak is configured differently. I didn't want two separate versions of Desktop files (one for Flatpak, the other for standard Linux), so I resolved it by editing.

File: .appdata.xml (hunk 1 above)
Comment (hfiguiere) [reply .appdata.xml:24]: the "desktop menu" is irrelevant here. It only relies on installed .desktop file.

File: .appdata.xml [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+<!-- Copyright 2023 Jakub Melka -->
+<component type="desktop-application">
+  <id>io.github.JakubMelka.Pdf4qt</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LGPL-3.0-only</project_license>
+  <name>PDF4QT</name>
+  <summary>Open Source PDF Editor</summary>
+  <description>
+    <p>
+      Unleash the power of PDF4QT: Your go-to open source PDF editor powered by the Qt framework. With a robust C++ library, intuitive PDF viewing/editing applications, and a handy command-line tool, PDF4QT streamlines your PDF interactions. Try it today.
```
Comment (hfiguiere) [inline .appdata.xml:11]: Mentions of `C++` and `Qt framework` are irrelevant to users.

File: screenshots/profi-addons.png
Comment (hfiguiere) [inline screenshots/profi-addons.png:1]: screenshots should be hosted on the application website instead.

File: build_pdf4qt.sh (hunk 6 above)
Comment (hfiguiere) [reply build_pdf4qt.sh:42]: no. that's just wrong. I maintain what I said. A .desktop file NEVER SHOULD reference an absolute path be they icons or executables.

Comment (hfiguiere) [reply build_pdf4qt.sh:42]: you also fail to honour `PREFIX`.

File: io.github.JakubMelka.Pdf4qt.json (hunk 2 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json:26]: It seems to be possible that VCPKG can run in offline mode, if packages are downloaded before. I will try it and update build system.

File: .appdata.xml (hunk 1 above)
Comment (JakubMelka) [reply .appdata.xml]: I have reduced these four launchables to a single one.

File: .appdata.xml [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+    <category>Office</category>
+    <category>Utility</category>
+  </categories>
+  <developer_name>Jakub Melka</developer_name>
+  <url type="homepage">https://jakubmelka.github.io/</url>
+  <url type="bugtracker">https://github.com/JakubMelka/PDF4QT/issues</url>
+  <url type="help">https://github.com/JakubMelka/PDF4QT/discussions</url>
+  <update_contact>Mgr.Jakub.Melka_AT_gmail.com</update_contact>
+  <content_rating type="oars-1.1"/>
+  <launchable type="desktop-id">io.github.JakubMelka.Pdf4qt.Pdf4QtViewerProfi.desktop</launchable>
```
Comment (hfiguiere) [inline .appdata.xml:24]: launcheable should match the flatpak id.

You need to figure out which application you want to be the default and make it soo.

File: io.github.JakubMelka.Pdf4qt.json [hunk 13]
```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "io.github.JakubMelka.Pdf4qt",
+    "runtime": "org.kde.Sdk",
+    "runtime-version": "6.5",
+    "sdk": "org.kde.Sdk",
+    "command": "/app/bin/Pdf4QtViewerProfi",
+    "build-options": {
+	"build-args": [ "--share=network" ]
+	},
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:9]:
```suggestion
```

this is not allowed and will always fail.

File: io.github.JakubMelka.Pdf4qt.json [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+    "finish-args": [
+        "--socket=x11",
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:11]: if it supports wayland then it's `fallback-x11`

File: io.github.JakubMelka.Pdf4qt.json [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc",
+        "--filesystem=xdg-config/kdeglobals:ro"
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:15]: why do you need access to this?

File: .appdata.xml
Comment (hfiguiere) [reply .appdata.xml:1]: whatever name it's installed as is the best.

File: .appdata.xml (hunk 11 above)
Comment (JakubMelka) [reply .appdata.xml]: Fixed.

File: .appdata.xml
Comment (JakubMelka) [reply .appdata.xml:1]: Moved to the upstream.

File: screenshots/profi-addons.png
Comment (JakubMelka) [reply screenshots/profi-addons.png:1]: Screenshot moved to the application website.

File: build_pdf4qt.sh (hunk 5 above)
Comment (JakubMelka) [reply build_pdf4qt.sh]: Desktop file modifications have been entirely eliminated, and the revised files have been pushed to the upstream.

**Comment by JakubMelka** [comment]
bot, build io.github.JakubMelka.Pdf4qt

File: io.github.JakubMelka.Pdf4qt.json (hunk 13 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json]: OK, removed.

File: build_pdf4qt.sh (hunk 10 above)
Comment (JakubMelka) [reply build_pdf4qt.sh]: OK, removed.

File: build_pdf4qt.sh (hunk 9 above)
Comment (JakubMelka) [reply build_pdf4qt.sh]: It is fixed now.

File: build_pdf4qt.sh (hunk 6 above)
Comment (JakubMelka) [reply build_pdf4qt.sh]: It should be fixed.

File: io.github.JakubMelka.Pdf4qt.json (hunk 2 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json:22]: I hope I have fixed it.

File: io.github.JakubMelka.Pdf4qt.json (hunk 15 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json]: I have deleted "--filesystem=xdg-config/kdeglobals:ro" - I have heard that some settings is here for Qt applications, but it is probably not needed.

Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json]: Other feature are needed - my software can render PDF pages on GPU.

File: io.github.JakubMelka.Pdf4qt.json (hunk 14 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json]: Fixed.

**Comment by JakubMelka** [comment]
bot, build io.github.JakubMelka.Pdf4qt

**Comment by JakubMelka** [comment]
bot, build io.github.JakubMelka.Pdf4qt

File: io.github.JakubMelka.Pdf4qt.json [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+        "/pdf4qt",
+        "/vcpkg"
+    ],
+    "modules": [ 
+        {
+            "name": "pdf4qt",
+            "buildsystem": "cmake-ninja",
+            "post-install": [
+                "echo 'Finishing BUILD - move plugins into right place'",
+                "./postbuild.sh"
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:23]: these command should be inlined. and the `echo` is totally unecessary.

File: io.github.JakubMelka.Pdf4qt.json [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+            ],
+            "config-opts": [
+                "-S /run/build/pdf4qt",
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DPDF4QT_INSTALL_QT_DEPENDENCIES=0",
+                "-DCMAKE_TOOLCHAIN_FILE=/run/build/pdf4qt/vcpkg/scripts/buildsystems/vcpkg.cmake",
+                "-DCMAKE_INSTALL_PREFIX=/app"
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:30]:
```suggestion
                "-DCMAKE_TOOLCHAIN_FILE=/run/build/pdf4qt/vcpkg/scripts/buildsystems/vcpkg.cmake"
```

File: io.github.JakubMelka.Pdf4qt.json [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+    "modules": [ 
+        {
+            "name": "pdf4qt",
+            "buildsystem": "cmake-ninja",
+            "post-install": [
+                "echo 'Finishing BUILD - move plugins into right place'",
+                "./postbuild.sh"
+            ],
+            "config-opts": [
+                "-S /run/build/pdf4qt",
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:26]:
```suggestion
```

File: build_pdf4qt.sh
Comment (hfiguiere) [inline build_pdf4qt.sh:1]: remove this file it's no longer used.

File: io.github.JakubMelka.Pdf4qt.json [hunk 19]
```diff
@@ -0,0 +1,53 @@
...
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DPDF4QT_INSTALL_QT_DEPENDENCIES=0",
+                "-DCMAKE_TOOLCHAIN_FILE=/run/build/pdf4qt/vcpkg/scripts/buildsystems/vcpkg.cmake",
+                "-DCMAKE_INSTALL_PREFIX=/app"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/JakubMelka/flathub/releases/download/v1/pdf4qt_vcpkg_loc.zip",
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:35]: this is not sources it seems
```
/run/build/pdf4qt/vcpkg/vcpkg: /run/build/pdf4qt/vcpkg/vcpkg: cannot execute binary file
```

This is the failure on aarch64.

**Comment by hfiguiere** [comment]
there is no `io.github.JakubMelka.Pdf4qt.desktop` to be found which is why it can't find icons either.

As I mentionned previous, you chose the default app, and that's what `io.github.JakubMelka.Pdf4qt.desktop` is for. The other use the other deskop entries.

**Comment by hfiguiere** [comment]
Alternatively setting the proper `launchable` in the appstream file **may** work.

File: build_pdf4qt.sh (hunk 7 above)
Comment (JakubMelka) [reply build_pdf4qt.sh:10]: Updated to cmake-ninja

File: build_pdf4qt.sh (hunk 8 above)
Comment (JakubMelka) [reply build_pdf4qt.sh]: Removed during update to cmake-ninja.

File: build_pdf4qt.sh
Comment (JakubMelka) [reply build_pdf4qt.sh:1]: Removed

File: io.github.JakubMelka.Pdf4qt.json (hunk 18 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json]: Applied.

File: io.github.JakubMelka.Pdf4qt.json (hunk 17 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json]: Applied

File: io.github.JakubMelka.Pdf4qt.json (hunk 16 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json]: Inlined, echo removed.

File: io.github.JakubMelka.Pdf4qt.json (hunk 19 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json:35]: It is a C++ package manager, which runs in offline mode and builds dependencies. I have disabled aarch64.

**Comment by JakubMelka** [comment]
Tomorrow, I will try to solve the problem you suggested - select one application. But how, for example, LibreOffice does it? They too have multiple applications. Thank you for any advice.

bot, build io.github.JakubMelka.Pdf4qt

File: io.github.JakubMelka.Pdf4qt.json (hunk 19 above)
Comment (hfiguiere) [reply io.github.JakubMelka.Pdf4qt.json:35]: This is very disappointing to disable aarch64 for no real reason.

**Comment by hfiguiere** [comment]
> Tomorrow, I will try to solve the problem you suggested - select one application. But how, for example, LibreOffice does it? They too have multiple applications. Thank you for any advice.

The answer is there:

> As I mentioned previously, you chose the default app, and that's what io.github.JakubMelka.Pdf4qt.desktop is for. **The other use the other deskop entries.**

**Comment by hfiguiere** [comment]
And according to the manifest the choice is:

> `"command": "Pdf4QtViewerProfi",`

File: flathub.json [hunk 20]
```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: how does one use this on a Raspberry Pi or Asahi Linux?

Comment (JakubMelka) [reply flathub.json:2]: Well, at least something than nothing. Maybe I will support other platforms later.

**Comment by hfiguiere** [comment]
```
Exporting share/icons/io.github.JakubMelka.Pdf4qt.Pdf4QtDocDiff.svg
Exporting share/icons/io.github.JakubMelka.Pdf4qt.Pdf4QtDocPageOrganizer.svg
Exporting share/icons/io.github.JakubMelka.Pdf4qt.Pdf4QtViewerLite.svg
Exporting share/icons/io.github.JakubMelka.Pdf4qt.Pdf4QtViewerProfi.svg
Exporting share/icons/io.github.JakubMelka.Pdf4qt.svg
```

this is the wrong location but that's not the failure.

The problem is
https://github.com/JakubMelka/PDF4QT/blob/f8455d5ea400cae8eec566bc6a8cd65ff2099ac9/Desktop/io.github.JakubMelka.Pdf4qt.appdata.xml#L24

`<launchable type="desktop-id">io.github.JakubMelka.Pdf4qt.Pdf4QtViewerProfi.desktop</launchable>`, which mean it'll use the icon defined in this desktop file which lead to not finding it in the buildbot checks.
See https://github.com/flathub/flathub/pull/4499#issuecomment-1722276715

File: io.github.JakubMelka.Pdf4qt.json [hunk 21]
```diff
@@ -0,0 +1,49 @@
...
+        "/vcpkg"
+    ],
+    "modules": [ 
+        {
+            "name": "pdf4qt",
+            "buildsystem": "cmake-ninja",
+            "post-install": [
+                "mkdir -p /app/lib/pdf4qt",
+                "mv -f /app/lib/*Plugin.so*  /app/lib/pdf4qt/",
+                "cp /app/share/icons/io.github.JakubMelka.Pdf4qt.Pdf4QtViewerProfi.svg /app/share/icons/io.github.JakubMelka.Pdf4qt.svg",
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:24]: svg go in `/app/share/icons/hicolor/scalable/apps`

File: io.github.JakubMelka.Pdf4qt.json [hunk 22]
```diff
@@ -0,0 +1,49 @@
...
+        "/pdf4qt",
+        "/vcpkg"
+    ],
+    "modules": [ 
+        {
+            "name": "pdf4qt",
+            "buildsystem": "cmake-ninja",
+            "post-install": [
+                "mkdir -p /app/lib/pdf4qt",
+                "mv -f /app/lib/*Plugin.so*  /app/lib/pdf4qt/",
```
Comment (hfiguiere) [inline io.github.JakubMelka.Pdf4qt.json:23]: this smell a bug in your build system.

File: io.github.JakubMelka.Pdf4qt.json (hunk 19 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json:35]: Well, I plan to fix it, however, first I want to implement x86_64 first.

File: .appdata.xml (hunk 12 above)
Comment (JakubMelka) [reply .appdata.xml]: Fixed.

File: io.github.JakubMelka.Pdf4qt.json (hunk 21 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json]: Fixed.

File: io.github.JakubMelka.Pdf4qt.json (hunk 22 above)
Comment (JakubMelka) [reply io.github.JakubMelka.Pdf4qt.json]: I hope I have fixed the bug.

**Comment by JakubMelka** [comment]
bot, build io.github.JakubMelka.Pdf4qt

**Pull request closed without merging**